<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeismicDataSelector</class>
 <widget class="QDialog" name="SeismicDataSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>73</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Inline:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbInline"/>
     </item>
     <item>
      <widget class="QSpinBox" name="sbIlineCount"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Crossline:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbXline"/>
     </item>
     <item>
      <widget class="QSpinBox" name="sbXlineCount"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="rbRandomLine">
       <property name="text">
        <string>Random Line</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Order: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbOrder1"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbOrder2"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbOrder3"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
